cmd/layotto/main.go

```diff
@@ -54,6 +55,7 @@ import (
 	"github.com/dapr/components-contrib/pubsub/azure/servicebus"
 	pubsub_gcp "github.com/dapr/components-contrib/pubsub/gcp/pubsub"
 	pubsub_hazelcast "github.com/dapr/components-contrib/pubsub/hazelcast"
+	pubsub_inmemory "github.com/dapr/components-contrib/pubsub/in-memory"
```

Comment:
Emmmmmmm where is this file?
Are u preparing to contribute it to dapr's components repo?
 I'm afraid it's hard to persuade them ,because dapr's startup process relies on docker,and with docker,it's easy to startup a redis,so in-memory pubsub is not that necessary......
I recommend that you put this component in `pkg/mock/components/pubsub` in Layotto for now, and then we can submit a new proposal in Dapr community trying to persuade them. If dapr merge it ,we can delete our own files